src/wallet/api/wallet.cpp

```diff
@@ -1574,6 +1579,13 @@ PendingTransaction *WalletImpl::createTransaction(const string &dst_addr, const
     return transaction;
 }
 
+PendingTransaction *WalletImpl::createTransaction(const string &dst_addr, const string &payment_id, optional<uint64_t> amount, uint32_t mixin_count,
```

Comment:
Perhaps this function can be moved back to line 1410 in order to not introduce a diff?

Reply:
https://patch-diff.githubusercontent.com/raw/monero-project/monero/pull/5518.patch

Reply:
Ah, the function itself got changed. disregard.